### DISCUSSION TITLE
Bottom of Remote Screen Cut Off

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#2008] [General]
Hello all,

I've noticed since the latest 0.19.x upgrades, that when I open a remote TRMM session to a remote computer, I'm noticing that the bottom part of the screen is constantly 'cut off' based on the current agent screen scaling setting.  

For example, when scaling is set to 75% due to a user having a 27"+ monitor, half of the task bar on their remote screen is simply cut off.  I've attached a screenshot for better understanding.  When I change scaling to 62.5%, the entire task bar is non visible.  Also when scaling is set to 62.5%, part of the right hand side of the screen is cut off too to where the "X" close button is missing/non viewable.

These display issues aren't noticeable at 100% or 87.5%.  At 50%, they are back to normal as well.  

But at 37.5%, 25%, and 12.5%, the bottom and right cut off's are also noticeable.  Thankfully those scaling resolutions are so small I can't use them, but I primary use the 75% or 62.5% for larger screens and it becomes a navigation problem on the clients screen.

I'd be curious to know if this is a setting, or an actual bug in the TRMM program.  Thanks!!

![Scaling Issue](https://github.com/user-attachments/assets/14d0af29-a05c-4f2f-8f46-5500b063042b)

**Comment by pdifeo** [comment]
Click on the arrow at rigth

![image](https://github.com/user-attachments/assets/fa529036-5ebc-4206-88ae-122d5e9e01fc)

**Comment by modem-19** [comment]
I tried that, but as it rotates through view modes, the bottom taskbar is still cut off / missing from view.

**Reply by pdifeo** [reply]
Not circle arrows, the last one on the rigth

![image](https://github.com/user-attachments/assets/ad84f5c8-cd01-4b19-9fb1-f12171a520c8)

**Comment by modem-19** [comment]
For some reason this didn't work on one specific computer, but did on another desktop I was using and seemed to solve the issue.

Thanks!
